src/test/java/com/github/cameltooling/lsp/internal/hover/CamelLanguageServerHoverTest.java

```diff
@@ -66,12 +66,21 @@ public void testDontProvideDocumentationOnHoverForBadPlaces() throws Exception {
 	
 	@Test
 	public void testDontProvideDocumentationOnHoverWhenEndingWithAnd() throws Exception {
-		CamelLanguageServer camelLanguageServer = initializeLanguageServer("<from uri=\"ahc:httpUri?test=test&\" xmlns=\"http://camel.apache.org/schema/spring\"></from>\n");
+		CamelLanguageServer camelLanguageServer = initializeLanguageServer("<from uri=\"ahc:httpUri?bufferSize=1024&\" xmlns=\"http://camel.apache.org/schema/spring\"></from>\n");
```

Comment:
why it has been changed?
it should work also when there is one attribute which is unknown